examples/contrib/oed/ab_test.py

```diff
@@ -0,0 +1,112 @@
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+
+import pyro
+import pyro.distributions as dist
+from pyro import optim
+from pyro.contrib.oed.eig import vi_ape
+
+
```

Comment:
Add a short expository description of the example here; see e.g. `examples/baseball.py`